package.json

```diff
@@ -48,6 +44,11 @@
           "type": "string",
           "default": "",
           "description": "Set text color for the filename in the status bar."
+        },
+        "ActiveFileInStatusBar.showFolderAsProjectName": {
```

Comment:
This should probably a boolean, and not be used for the splitting of the path above - since that is not intuitive.